### PR TITLE
feat :  Archive page

### PR DESCRIPTION
**Pull request by Tanjum786**
added archive page 
- move to archive 
- delete from the archive 
- restore from archive

### CODE REVIEW

**Review by nishasen — APPROVED** [review]
PR looks clean and good, I can see how you optimised some of the code.

**Review by shubhamkumar648 — APPROVED** [review]
everything looks good. i like the way use variable name which helps easy to read

File: src/Hooks/Reducer/notesReducerFun.jsx [hunk 1]
```diff
@@ -1,11 +1,62 @@
...
+      return {
+        ...NotesState,
+        NotesList: [...NotesList, NotesAction.payload],
+        archiveList:DleteArchive
+      };
+    case "DELETE_FROM_ARCHIVE":
+      return {
+        ...NotesState,
+        archiveList: archiveList.filter(
+          (item) => item.id != NotesAction.payload.id
```
Comment (Omkar-Chaskar) [inline src/Hooks/Reducer/notesReducerFun.jsx:55]: You can add one more line 
trashList: [...trashList, NotesAction.payload], 
to add note to trash. so you can restore it

**Review by Omkar-Chaskar — APPROVED** [review]
Good to merge . Just small suggestion. You can remove  Note button from side component and Give that link to Home  button.
On clicking Home button it redirect to landing page

**Pull request merged**